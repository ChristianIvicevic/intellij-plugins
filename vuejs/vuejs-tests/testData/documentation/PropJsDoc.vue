<script>
export default {
  props: {
    /** @type {import('vue').PropType<{strVar: string | string[]}>} */
    pr<caret>op1: Object,
  }
}
</script>