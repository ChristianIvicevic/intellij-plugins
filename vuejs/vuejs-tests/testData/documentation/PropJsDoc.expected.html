<div class='definition'><pre>prop1: <span style="">{strVar:&#32;(</span><span style="color:#000080;">string&#32;</span><span style="">|&#32;</span><span style="color:#000080;">string</span><span style="">[])}</span></pre></div><table class='sections'><tr><td valign='top'><icon src='AllIcons.FileTypes.JavaScript'/>&nbsp;PropJsDoc.vue</td></table>